test/new-e2e/tests/agent-subcommands/subcommands_test.go

```diff
@@ -151,14 +151,13 @@ func (v *subcommandSuite) TestDefaultInstallStatus() {
 			shouldBePresent: true,
 		},
 		{
-			name:            "JMXFetch",
+			name:            "JMX Fetch",
 			shouldBePresent: true,
 			shouldContain:   []string{"no checks"},
 		},
 		{
 			name:            "Logs Agent",
-			shouldBePresent: true,
-			shouldContain:   []string{"Logs Agent is not running"},
+			shouldBePresent: false,
```

Comment:
In the newest version, the Logs Agent section will not appear if the logs agent is not enabled. I the previous version, the section would be displayed with the message `"Logs Agent is not running"`

Here is the code that injects the log agent status provider https://github.com/DataDog/datadog-agent/blob/1f83d846d513802965498d7a38987142468805dd/comp/logs/agent/agent.go#L121

Reply:
Are we sure it's on purpose ?

Reply:
Anything in the status output is very legacy. So, it is hard to know what is on purpose or not. 

My two cents:
The status output is enormous, so sections telling you something is not running take up screen space. So, if the logs agent is enabled, we should not show anything. 

cc @gh123man

Reply:
I could se an argument that the old message was actually useful: I've run into situations where I run `agent status` and seeing the message about some feature being disabled or turned off let's me know to stop looking for it. With the new version I have to look through the entire output to see that a specific feature does *not* show up anywhere. I realize this PR is just fixing the test, but was wondering how hard it might be in the code to change this behavior to do something similar to what was happening before.

Reply:
I think this is ok. It should be pretty clear from `agent config` if the logs agent is disabled, or from the agent logs if an error occurred.

Reply:
Even if it doesn't seem like it, I think there's somewhat of consistency here in UX expectations which is pretty much: "it's ok to skip this section if the user clearly knows not to look for it". I'd probably add that using multiple commands to know if the status output is correct seems like a bad end-user workflow.

With that in mind, the only place where you can see as part of the same command output that logs are enabled is a line in the `Metadata` section and it's not very obvious to someone not familiar with what they're looking for. Because of that, I'd say that we should either keep this but make enablement flags more prominent or stick with the old `Logs Agent is not running` section output.